R/prep_data.R

```diff
@@ -59,10 +59,18 @@
 #'   into a new category, `other`. If numeric, must be in {0, 1}, and is the
 #'   proportion of observations below which levels will be grouped into other.
 #'   See `recipes::step_other`.
+#' @param PCA Integer or Logical. PCA is done to reduce training time,
```

Comment:
Don't forget to add PCA to the bulleted list at the start of the prep data docs.